src/assemblyItemsBrowser.js

```diff
@@ -108,11 +109,17 @@ module.exports = React.createClass({
 		return ret
 	},
 
-	resolveAddress: function(address)
+	resolveAddress: function(address, props)
 	{
 		if (!this.state.codes[address])
 		{
-			var hexCode = web3.eth.getCode(address)	
+			console.log("loading new code from web3 " + address)
+			var hexCode
+			if (this.state.currentSelected === -1 && props.transaction.to === null) // contract creation
+				hexCode = props.transaction.input
+			else
+				hexCode = web3.eth.getCode(address)
+			
 			var code = codeUtils.nameOpCodes(new Buffer(hexCode.substring(2), 'hex'))
 			this.state.codes[address] = code[0]
```

Comment:
This is kind of wrong, because the code will not be stored at that address for a creation transaction. Can we create a new field for the currently executing code?